deisctl/utils/utils_test.go

```diff
@@ -18,3 +18,24 @@ func TestResolvePath(t *testing.T) {
 		}
 	}
 }
+
+func TestGetEnvUint(t *testing.T) {
+	defaultValue := uint64(3)
+	goodVars := []string{"1", "2", "4"}
+	expectedGoodUints := []uint64{1, 2, 4}
+	badVars := []string{"-1", "abc"}
+	for index, goodVar := range goodVars {
+		os.Setenv("DEIS_NUM_ROUTERS", goodVar)
+		result := GetEnvUint("DEIS_NUM_ROUTERS", defaultValue)
+		if expectedGoodUints[index] !=  result {
```

Comment:
The extra space here will fail `make -C deisctl test-style`, since we expect all .go code to have `gofmt` run on it.